src/App.tsx

```diff
@@ -1,10 +1,20 @@
 import { GlobalStyle } from "./GlobalStyles";
-import { Navbar } from "./components/index";
+import { Modal, Navbar } from "./components/index";
```

Comment:
No need to use `/index` it is assumed to be `index/js` automatically

Reply:
done